sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -200,6 +200,42 @@ public void fillInWebElement(By selector, String text) {
     }
   }
 
+  public void fillInAndLeaveWebElement(By selector, String text) {
```

Comment:
looks like duplicated code.
Just use: fillInWebElement()  and your needed line: baseSteps.getDriver().findElement(By.cssSelector("body")).sendKeys(Keys.TAB);  
and maybe a click inside if needed

Reply:
That was my first approach to this matter as well, but unfortunately it doesn't work.
The TAB needs to be sent from inside the fillInWebElement() method (or rather its copy), not after it. Otherwise the test gets stuck inside the text input field, unable to finish executing fillInWebElement(), and eventually failing through a timeout.

Reply:
ok, and if we create a separated method to send the keys? like, using the fill, then a method to move outside field by sending the tab

Reply:
In such case the second method will not execute at all, because the test gets stuck and timeouts before the first one finishes. Wthout the Tab to exit the text input field, fillInWebElement() doesn't execute all the way to the end.